kahuna/public/js/components/gr-panel/gr-panel.js

```diff
@@ -44,10 +48,28 @@ grPanel.controller('GrPanel', [
 
         var ctrl = this;
 
+        const panelName = 'gr-panel';
+
+        panelService.addPanel(panelName, false);
+        ctrl.isVisible = panelService.isVisible(panelName);
+
+        $rootScope.$on(
+            'ui:panels:' + panelName + ':updated',
```

Comment:
FYI in general I'm not keen on dynamically generated keys for things like that because it's hard/impossible to find where things are used (if static, can just use grep). Not a biggie, but if it's always the same here, might as well just make the string static in the source?